src/tools/create-require.js

```diff
@@ -0,0 +1,21 @@
+'use-strict';
+
+const { dependencies, devDependencies } = require('../../package.json');
+const { RequireModuleError } = require('../errors');
+
+const createRequire = () => module => {
```

Comment:
While not a reserved word `module` is already defined (eg `module.exports`) and we probably shouldn't reuse it

Reply:
Ha! That's an excellent point. 💯